randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1097,7 +1114,13 @@ Extra - asset_id: collision_camera_010
   > Door to EMMI Zone Exit Southwest (DoorFrame_005)
       Trivial
   > Door to Wide Beam Block Room (DoorFrame_007)
-      Simple IBJ or Use Spin Boost
+      Speed Booster or Simple IBJ or Use Spin Boost
+  > Pickup (Power Bomb Tank)
+      All of the following:
+          Morph Ball and Speed Booster
+          Any of the following:
+              # doing this spark while the emmi is there is easier said than done
+              After Artaria - Central Unit or Movement (Intermediate)
```

Comment:
I believe it was discussed that this spark is fine while the emmi is there, but not any of the other options to reach the item (other than IBJ)